content/config/completion.md

```diff
@@ -0,0 +1,96 @@
+---
+title: CLI completion
+description: Set up your shell to get autocomplete for Docker commands and flags
+keywords: cli, shell, fish, bash, zsh, completion, options
+---
+
+You can generate a shell completion script for the Docker CLI using the `docker
+completion` command. The completion script gives you word completion for
+commands, flags, and Docker objects (such as container and volume names) when
+you hit `<Tab>` as you type into your terminal.
+
+You can generate completion scripts for the following shells:
+
+- [Bash](#bash)
+- [Zsh](#zsh)
+- [fish](#fish)
+- PowerShell
+
+## Bash
+
+Bash does not support completions natively, and requires the `bash-completion` package to be installed.
+
```

Comment:
I was wrong to say `bash` does not support completions natively.

https://www.gnu.org/savannah-checkouts/gnu/bash/manual/bash.html#Programmable-Completion

 https://tuzz.tech/blog/how-bash-completion-works#how-completion-functions-work

But it's a cumbersome setup. So we just simplify this to, to get completions working correctly use `bash-completion`.